### ISSUE TITLE
Invoke-GetCommand : Exception calling "DownloadString" with "1" argument(s): "The request was aborted: Could not create SSL/TLS secure channel."

### ISSUE THREAD

**Issue by RatheeshPG** [rfennell/AzurePipelines#787]
Hi Team,

I am new to ADO and we have a task based on "Generate Release Notes for Pipeline Builds or Releases (PowerShell)", the task is working till date from past couple of months. suddenly today we are getting below exception. could you please help us what is causing this . 

Invoke-GetCommand : Exception calling "DownloadString" with "1" argument(s): "The request was aborted: Could not create SSL/TLS secure channel."

System.Management.Automation.MethodInvocationException: Exception calling "DownloadString" with "1" argument(s): "The request was aborted: Could not create SSL/TLS secure channel." ---> System.Net.WebException: The request was aborted: Could not create SSL/TLS secure channel.
2020-08-12T13:49:32.9599458Z ##[debug]   at System.Net.WebClient.DownloadDataInternal(Uri address, WebRequest& request)
2020-08-12T13:49:32.9609620Z ##[debug]   at System.Net.WebClient.DownloadString(Uri address)
2020-08-12T13:49:32.9619947Z ##[debug]   at CallSite.Target(Closure , CallSite , Object , Object )
2020-08-12T13:49:32.9630125Z ##[debug]   --- End of inner exception stack trace ---
2020-08-12T13:49:32.9640345Z ##[debug]   at System.Management.Automation.ExceptionHandlingOps.CheckActionPreference(FunctionContext funcContext, Exception exception)
2020-08-12T13:49:32.9650541Z ##[debug]   at System.Management.Automation.Interpreter.ActionCallInstruction`2.Run(InterpretedFrame frame)
2020-08-12T13:49:32.9660778Z ##[debug]   at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
2020-08-12T13:49:32.9670990Z ##[debug]   at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
2020-08-12T13:49:32.9681223Z ##[debug]   at System.Management.Automation.Interpreter.Interpreter.Run(InterpretedFrame frame)
2020-08-12T13:49:32.9692111Z ##[debug]   at System.Management.Automation.Interpreter.LightLambda.RunVoid1[T0](T0 arg0)
2020-08-12T13:49:32.9702418Z ##[debug]   at System.Management.Automation.PSScriptCmdlet.RunClause(Action`1 clause, Object dollarUnderbar, Object inputToProcess)
2020-08-12T13:49:32.9712504Z ##[debug]   at System.Management.Automation.PSScriptCmdlet.DoEndProcessing()
2020-08-12T13:49:32.9722484Z ##[debug]   at System.Management.Automation.CommandProcessorBase.Complete()

**Comment by rfennell** [comment]
I have just retested this extension with my test harnesses and cannot repeat the issue you reported. I retested using both a private agent and an Azure hosted agent.

I know Microsoft were planning to make some changes over TLS that could cause this type of problem for PowerShell based tasks, but they [postponed the change](https://devblogs.microsoft.com/devops/azure-devops-services-to-require-tls-1-2/).

If you using a private build agent, have you made any changes to your firewall or operating system settings concerning SSL/TLS handling?

It is not a fix for the task you are using, but I would recommend you consider swap to using my [node based cross platform release notes task](https://marketplace.visualstudio.com/items?itemName=richardfennellBM.BM-VSTS-XplatGenerateReleaseNotes). I plan to make no enhancement to this PowerShell version and will probably deprecate it in the next 12 months.
